### PR TITLE
sks: create: use the latest version by default

### PR DESCRIPTION
**Pull request by falzm**
This changes modifies the `exo sks create` command to use the latest
available version by default, instead of hardcoding it.

### CODE REVIEW

File: cmd/sks_create.go [hunk 1]
```diff
@@ -88,6 +84,22 @@ Supported output template annotations: %s`,
...
+		if version == "latest" {
+			ctx := exoapi.WithEndpoint(gContext, exoapi.NewReqEndpoint(gCurrentAccount.Environment, zone))
+			versions, err := cs.ListSKSClusterVersions(ctx)
+			if err != nil || len(versions) == 0 {
+				if len(versions) == 0 {
+					err = errors.New("no version returned by the API")
+				}
+				return fmt.Errorf("unable to retrieve SKS versions: %s", err)
+			}
+			version = versions[0]
```
Comment (zyegfryed) [inline cmd/sks_create.go:100]: What do you think about extracting that code to a dedicated function? This way it should be simple to test it.

Comment (falzm) [reply cmd/sks_create.go:100]: I'm not sure it's worth testing this specific part, since it's mostly egoscale code which already tests it...

**Review by zyegfryed — APPROVED** [review]

**Review by sgmac — APPROVED** [review]

**Comment by sgmac** [comment]
We should merge asap as this is currently failing to create SKS clusters

```shell
 ✔ Creating SKS cluster "sks-conformance-gva2"... 0s
error: invalid request: Invalid Kubernetes version 1.20.2. Available versions are: 1.20.5
```

**Comment by falzm** [comment]
> We should merge asap as this is currently failing to create SKS clusters

You can always specify a version e.g. `--version=1.20.5`.

**Review by mcorbin — APPROVED** [review]

**Pull request merged**